packages/jaeger-ui/src/components/App/Page.tsx

```diff
@@ -52,7 +52,7 @@ export class PageImpl extends React.Component<TProps> {
 
   render() {
     const { embedded } = this.props;
-    const contentCls = cx({ 'Page--content': !embedded });
+    const contentCls = cx({ 'Page--content': true });
```

Comment:
I do not have enough context to understand the impact of this change. It looks like there was an explicit decision NOT to show this in the embed mode, there must have been a reason for that.

Reply:
cc @rubenvp8510

Reply:
when i directly set the classname to Page-content, it'll show an nav height blank on top of window, and when scroll the page,the layout displayed messy ,so i add an new classname copyed from 'Page--content' ,just delete the "top: var(--nav-height);" property,it seem's ok.